packages/avatars/src/elements/Avatar.tsx

```diff
@@ -81,7 +84,9 @@ Avatar.defaultProps = {
 
 (Avatar as any).Text = StyledText;
 
-/** @component */
+/** @component
```

Comment:
I meant to mention, you can remove `@component` tags as you encounter them. They are no longer relevant for documentation.

Reply:
I'll keep that in mind as I go through the other packages. I've removed it in the latest update